django_celery_results/migrations/0013_alter_taskresult_status.py

```diff
@@ -0,0 +1,23 @@
+# Generated by Django 5.0.6 on 2024-06-19 09:26
```

Comment:
you can delete this file and regenerate a new one with django 5.1.x

Reply:
Done!